java/spring/webapp/ArmTemplates/windows-webapp-template.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
@hiyadav  @sachinma Can we use symbolic links rather than duplicating the file everywhere? AzurePrep can replace links with actual files while pushing code to created project.